influxql/parser.go

```diff
@@ -1435,8 +1435,8 @@ func (p *Parser) parseOptionalTokenAndInt(t Token) (int, error) {
 	// Parse number.
 	n, _ := strconv.ParseInt(lit, 10, 64)
 
-	if n < 1 {
-		msg := fmt.Sprintf("%s must be > 0", t.String())
+	if n < 0 {
```

Comment:
any reason to validate this as all? Wouldn't it be better to just look at the error from `ParseInt`? I'm guessing that's why it wasn't allowing zero before.